.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
```

Comment:
I think you pushed some stuff you shouldn't have? This file shouldn't be here I don't think